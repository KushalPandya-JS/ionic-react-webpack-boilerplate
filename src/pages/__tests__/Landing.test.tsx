import React from 'react';
import renderer from 'react-test-renderer';
import { Landing } from '../landing';

describe('Landing', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Landing />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});