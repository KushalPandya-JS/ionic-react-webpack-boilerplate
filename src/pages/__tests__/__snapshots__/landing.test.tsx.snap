// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapshot renders 1`] = `
<div
  className="ion-page"
>
  <ion-header>
    <ion-toolbar>
      <ion-title>
        Landing
      </ion-title>
    </ion-toolbar>
  </ion-header>
  <ion-content>
    Content
  </ion-content>
</div>
`;
